src/drive/mobile/lib/media.js

```diff
@@ -141,3 +142,24 @@ export const getPhotos = async () => {
 
   return Promise.resolve(defaultReturn)
 }
+
+export const getMediaBuckets = async () => {
```

Comment:
We should only call this method on Android since `listMediaBuckets` is not available on iOS

Reply:
Fixed

https://github.com/cozy/cozy-drive/compare/5da40f02a715ac92fd0b811a52b5f9e8536dc879..8bbeeace9e475c88b9c93ef8ab34263c9c572e9f